### DISCUSSION TITLE
Programmatic way to export bib file from Zotero

### DISCUSSION THREAD

**Discussion by wfondrie** [retorquere/zotero-better-bibtex#2162] [Q&A] [answered]
Hi all 👋 ,

I'm looking for a way to programmatically export a .bib file from a Zotero library. 

I'd like to setup a system with GitHub actions that automatically builds a .bib file for a Zotero Group Library, so that my team has a single .bib file we can reference when writing. 

Has anyone setup a system like this? Does it work well?

**Comment by retorquere** [comment, marked as answer]
If you are satisfied with the stock bibtex export, you can use the zotero web API. If you want BBT exports, it is possible but not trivial. My own tests run as github actions, running the actual Zotero client under xvfb, driving it's behavior using [this plugin](https://github.com/retorquere/zotero-better-bibtex/tree/master/test/fixtures/debug-bridge).

There is no all, just me.

**Reply by wfondrie** [reply]
Thanks @retorquere, both for the helpful reply and for maintaining this great tool.
